test/BUILD

```diff
@@ -776,22 +776,22 @@ scala_junit_test(
     runtime_deps = [":JunitMixedSeparateTarget"],
 )
 
-scala_library(
-    name = "JunitRuntimePlatform",
-    srcs = [
-        "src/main/scala/scalarules/test/junit/runtime_platform/JunitRuntimePlatformTest.java",
-    ],
-    deps = ["@io_bazel_rules_scala_junit_junit"],
-)
-
-scala_junit_test(
-    name = "JunitRuntimePlatform_test_runner",
-    size = "small",
-    runtime_jdk = "@bazel_tools//tools/jdk:remote_jdk11",
-    suffixes = ["Test"],
-    tests_from = [":JunitRuntimePlatform"],
-    runtime_deps = [":JunitRuntimePlatform"],
-)
+#scala_library(
+#    name = "JunitRuntimePlatform",
+#    srcs = [
+#        "src/main/scala/scalarules/test/junit/runtime_platform/JunitRuntimePlatformTest.java",
+#    ],
+#    deps = ["@io_bazel_rules_scala_junit_junit"],
+#)
+#
+#scala_junit_test(
+#    name = "JunitRuntimePlatform_test_runner",
+#    size = "small",
+#    runtime_jdk = "@bazel_tools//tools/jdk:remote_jdk11",
+#    suffixes = ["Test"],
+#    tests_from = [":JunitRuntimePlatform"],
+#    runtime_deps = [":JunitRuntimePlatform"],
+#)
```

Comment:
@simuons are these accidentally commented out, or is there a reason? If there's a reason, please leave a comment in the code.

Reply:
@liucijus thanks for noting. I forgot about it.

Reply:
@liucijus fixed test for jdk > 11. Please take a look.